messenger.go

```diff
@@ -116,6 +123,65 @@ func (m *Messenger) ProfileByID(id int64) (Profile, error) {
 	return p, err
 }
 
+// GreetingSetting sends settings for greeting
+func (m *Messenger) GreetingSetting(text string) error {
+	d := GreetingSetting{
+		SettingType: "greeting",
+		Greeting: GreetingInfo{
+			Text: text,
+		},
+	}
+
+	data, err := json.Marshal(d)
```

Comment:
Was there a particular reason you chose to use `json.Marshal` here instead of something like the following:

```
	var buf bytes.Buffer

	err := json.NewEncoder(&buf).Encode(pr)
	if err != nil {
		return err
	}
```

This would allow us to pass `buf` directly to `http.NewRequest`.